src/components/ui/AuthModal/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,81 @@
+import { Button, Stack, TextField } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import getValidationSchema from './validationSchema';
+
+type TSignInProps = {
+    closeModal: () => void;
+};
+
+const initialValues = {
+    email: '',
+    password: '',
+};
+
+const SignIn = ({ closeModal }: TSignInProps) => {
+    const { t } = useTranslation('auth');
+
+    const handleSubmit = () => {
+        console.log('Sign in submit');
```

Comment:
Could remove logs to console when submit functionality is implemented.

Reply:
I have updated it in the recent commit.